ruby/whatsapp-with-vonage/lib/main.rb

```diff
@@ -0,0 +1,71 @@
+require 'jwt'
+require 'httparty'
+require 'json'
+require 'digest/sha2'
+require 'dotenv/load'
```

Comment:
Were you using this for testing?

Shouldn't be necessary for the template

Reply:
hi @loks0n , i still confused about which `require` that you mentioned above that is (you assumed that i) used for testing?

Reply:
I'm referring to 'dotenv/load', we don't need this as the open runtime container does this for automatically.

Reply:
ah i see @loks0n , okay will fix it immediately
thank you!